lib/quiz/match_the_following.dart

```diff
@@ -0,0 +1,182 @@
+import 'package:flutter/material.dart';
+
+Map<String, dynamic> _selected = {};
```

Comment:
We should **never** have global variables

Reply:
Global variable removed.